saskatoon/sitebase/views.py

```diff
@@ -7,11 +7,29 @@
 from django.views.generic import View, TemplateView
 from harvest.models import Harvest, RequestForParticipation
 from saskatoon.settings import EQUIPMENT_POINTS_PDF_PATH, VOLUNTEER_WAIVER_PDF_PATH
+from sitebase.models import Content
+
+VOLUNTEER_HOME_CONTENT_NAME = 'volunteer_home'
+PICKLEADER_HOME_CONTENT_NAME = 'pickleader_home'
 
 
 class Index(TemplateView):
     template_name = 'app/index.html'
 
+    def get_context_data(self, **kwargs):
+        context = super().get_context_data(**kwargs)
+
+        content_name = VOLUNTEER_HOME_CONTENT_NAME
+
+        if self.request.user.is_authenticated:
+            content_name = PICKLEADER_HOME_CONTENT_NAME
+
+        home = Content.objects.get(name=content_name)
+        if home is not None:
+            context['content'] = home.content(self.request.LANGUAGE_CODE)
+
```

Comment:
TODO: pass a dummy content object here if there are no appropriate object saved in db

Reply:
my lsp is saying `Content.objects` doesn't exist. as it stands, hitting `/` route gives this error. wrapping the method in try/catch and manually setting `home = None` on exception at least bypasses this, but original problem remains and `get()` may never get called. not sure where to go from there.

Reply:
have you run migrations? `saskatoon/manage.py migrate`

Reply:
pushed new commit, now if the `Content` object with reference name `pickleader_home`/`volunteer_home` doesn't exist when first accessing the home page, a blank object is created.